models/user.js

```diff
@@ -0,0 +1,43 @@
+const Joi = require('joi')
+const { Schema, model } = require('mongoose')
+const emailPattern = /^[a-zA-Z0-9._%+-]+@[a-zA-Z0-9.-]+\.[a-zA-Z]{2,}$/
```

Comment:
you can separate all the paterns in separated file, and just import them, where you need. It's optional